obal/data/playbooks/nightly/nightly.yaml

```diff
@@ -8,14 +8,14 @@
   tasks:
     - name: 'legacy nightly building'
       block:
-        - name: 'set nightly_releaser'
```

Comment:
Why break this? Modern building shouldn't use this.

Reply:
I am not following you.

Reply:
Is there a reason you're removing this code? As I understand it the new way of building shouldn't hit this code so it should be safe to keep it in.

Reply:
Another thought: perhaps move it into a role and write all the legacy building in one file while doing the modern way in another file. IMHO it's easier to follow than huge blocks with when statements.

Reply:
the new way of building shouldn't hit this, correct. but it's redundant to have for legacy nightly building.

RE: roles
that's also a good idea and i agree with you, but sounds more like refactoring work.

Reply:
This task has been returned.